pyart/graph/gridmapdisplay.py

```diff
@@ -74,14 +74,14 @@ def __init__(self, grid, debug=False):
         self.debug = debug
 
         # set up the projection
-        lat0 = grid.axes['lat']['data'][0]
-        lon0 = grid.axes['lon']['data'][0]
+        lat0 = grid.origin_latitude['data'][0]
+        lon0 = grid.origin_longitude['data'][0]
         self.proj = pyproj.Proj(proj='aeqd', datum='NAD83',
```

Comment:
**GridMapDisplay** needs to be updated to use the new **Grid** attributes.  This is on my to-do list but I was going to implement these changes in a separate PR.  My plan is that the **GridMapDisplay** will make use of the **Grid** projection, but I think it can wait until this later PR.